src/Codeception/Test/Cest.php

```diff
@@ -155,7 +155,11 @@ public function toString()
 
     public function getSignature()
     {
-        return get_class($this->getTestClass()) . ":" . $this->getTestMethod();
+        $signature = get_class($this->getTestClass()) . ":" . $this->getTestMethod();
+        if (!empty($this->getMetadata()->getCurrent('iteration'))) {
```

Comment:
`!empty(expr)` does not work on PHP 5.4,  please assign the result to temporary variable.